MvvmCross/Windows/Wpf/Views/Presenters/MvxWpfViewPresenter.cs

```diff
@@ -19,35 +19,56 @@
 namespace MvvmCross.Wpf.Views.Presenters
 {
     public class MvxWpfViewPresenter
-        : MvxBaseWpfViewPresenter
+        : MvxBaseWpfViewPresenter, IMvxAttributeViewPresenter
```

Comment:
@jz5 can you check the changes here?

Reply:
WPF playground doesn't work expectedly. (WindowView and WindowChildVIew)
Because, override PresentationAttribute is called so early. The view can't refer ViewModel.
WindowView(Model) and WindowChildVIew(Model) are for multiple parent windows and multiple child contents. (Please run the previous playground)
I think that it is good if there is the workaround for View/ViewModel.

Reply:
The problem is that it was never intended to work with ViewModel because ViewModel shouldn't handle view state. What workaround are you thinking of?

Reply:
In this case, I think that View handle state using ViewModel state.
However, I will consider the workaround.